app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -812,8 +812,8 @@ def prov_set_form_vars(req = nil)
         @edit[:new][:schedule_time] = format_timezone(@edit[:new][:schedule_time], Time.zone, "raw")
         @edit[:new][:start_date] = "#{@edit[:new][:schedule_time].month}/#{@edit[:new][:schedule_time].day}/#{@edit[:new][:schedule_time].year}" # Set the start date
         if params[:id]
-          @edit[:new][:start_hour] = "#{@edit[:new][:schedule_time].hour}"
-          @edit[:new][:start_min] = "#{@edit[:new][:schedule_time].min}"
+          @edit[:new][:start_hour] = (@edit[:new][:schedule_time].hour).to_s
```

Comment:
Thanks, I'll fix them in here.  `rubocop -a` fixes it like that.  I manually removed the superfluous `()` in the handful of places I found it.